worldeditadditions/utils/node_identification.lua

```diff
@@ -12,7 +12,7 @@ function worldeditadditions.is_airlike(id)
 	elseif id == node_id_ignore then -- ignore = not loaded yet IIRC (so it could be anything)
 		return false
 	end
-	
+
```

Comment:
I'm sorry, but I can't merge this if the whitespace on these lines is deleted. Can you restore the whitespace here and on similar lines please?

It's rather annoying in my code editor when I want to edit that line.

Reply:
Really! No problem, usually that's not desired.  But I'll fix it 👍🏻

Reply:
Thanks :D

Yeah, I'm somewhat unusual lol